experimental/sdg-tracker/src/components/countries/CountriesContent.tsx

```diff
@@ -204,6 +228,59 @@ function addTileToHierarchy(
   }
 }
 
+/**
+ * builds object to store tiles in order of display 
+ */
+function buildTileHierarchy(
+  chartConfigCategory: ChartConfigCategory,
+  mainTopics: RelatedTopic[],
+  selectedTopics: string[],
+  varToTopics: VarToTopicMapping,
+): TileHierarchy {
+  // stores hierarchy of Goals -> Target -> Indicator -> Tiles
+  const hierarchy: Goals = {};
+  // stores tiles in order returned by fulfillment api
+  const orderedTiles: ChartConfigTile[] = [];
+  // stores topic dcids covered by the tiles
+  const topicDcids: string[] = [];
+
+  // iterate over tiles nested in chartConfigCategory
+  chartConfigCategory.blocks.forEach((block) => {
+    block.columns.forEach((column) => {
+      column.tiles.forEach((tile) => {
+        if (tile.type === "PLACE_OVERVIEW") {
+          return;
+        }
+        if (_.isEmpty(tile.statVarKey)) {
+          return;
+        }
+        const statVarKey = tile.statVarKey[0];
+        if (_.isEmpty(chartConfigCategory.statVarSpec[statVarKey])) {
+          return;
+        }
+        const statVar = chartConfigCategory.statVarSpec[statVarKey].statVar;
+        if (_.isEmpty(varToTopics[statVar])) {
+          return;
+        }
+        for (const topic of varToTopics[statVar]) {
+          addTileToHierarchy(tile, hierarchy, topic.dcid, selectedTopics);
+        }
+        orderedTiles.push(tile);
+        varToTopics[statVar].forEach((topic) => topicDcids.push(topic.dcid));
+      });
+    });
+  });
+
+  let topicNameStr = "";
+  topicDcids.sort();
+  if (mainTopics.length == 2) {
+    topicNameStr = `${mainTopics[0].name} vs. ${mainTopics[1].name}`;
+  } else if (mainTopics.length == 1) {
```

Comment:
Can you make this an `} else {` clause with no `mainTopics.length === 1` conditional ? That way it will trigger even if mainTopics.length > 2 (in case we support that in the future)

Reply:
Done!